.gitignore

```diff
@@ -146,3 +146,5 @@ dmypy.json
 
 # Pycharm
 .idea/
+
+script.py
```

Comment:
?

Reply:
Good spot, this was to stop me accidentally commiting the script I was using to try out changes. Will take it back out of the gitignore now I'm done though